content/docs/user-guide/networking/multiplayer/spawning.md

```diff
@@ -27,8 +29,15 @@ The `OnPlayerLeave` method of `IMultiplayerSpawner` provides a hook so that when
 
 `OnPlayerLeave` also takes a disconnect reason which allows responding to different kinds of disconnects. For example, it may be undesirable to clean up objects if a player times out if they can attempt to reconnect to the session.
 
-### Spawning examples in MultiplayerSample
 
-A practical example of an implementation for a spawner is [`MultiplayerSampleSystemComponent`](https://github.com/o3de/o3de-multiplayersample/blob/development/Gem/Code/Source/MultiplayerSampleSystemComponent.cpp) in the  [MultiplayerSample project](https://github.com/o3de/o3de-multiplayersample/). MultiplayerSample implements a "round robin"-style spawning system that gathers entities with `NetworkPlayerSpawnerComponents`. `MultiplayerSampleSystemComponent` then queries that system during `OnPlayerJoin`. `OnPlayerLeave` simply marks the entity passed in for removal. 
+## Examples in MultiplayerSample Project
+
+A practical example of an implementation for a spawner is [`MultiplayerSampleSystemComponent`](https://github.com/o3de/o3de-multiplayersample/blob/2c84827ffb20082b8c16fc0edc65cd49226f3cd2/Gem/Code/Source/MultiplayerSampleSystemComponent.cpp) in the  [MultiplayerSample project](https://github.com/o3de/o3de-multiplayersample/). MultiplayerSample Project implements a "round robin"-style spawning system that gathers entities with `NetworkPlayerSpawnerComponents`. `MultiplayerSampleSystemComponent` then queries that system during the `OnPlayerJoin` event. The `OnPlayerLeave` event simply marks the entity that's passed in for removal. 
+
+For more information about the MultiplayerSample project, refer to the [MultiplayerSample README](https://github.com/o3de/o3de-multiplayersample/blob/development/README.md).
```

Comment:
```suggestion
For more information about MultiplayerSample Project, refer to the [MultiplayerSample README](https://github.com/o3de/o3de-multiplayersample/blob/development/README.md).
```
Updated reference to match the title of the `README.md`, per our conversation in [this change suggestion thread](https://github.com/o3de/o3de.org/pull/2244#discussion_r1112661643).